### PR TITLE
Fixes Assunzione robe pixels

### PR DESCRIPTION
**Pull request by feartheblackout**
Credits to Geeves for the code that originated from the Dominian cape fix.

### CODE REVIEW

**Comment by feartheblackout** [comment]
!review

**Comment by feartheblackout** [comment]
!bugfix

File: html/changelogs/SimpleMaroon-assunzionerobething.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - bugfix: The pixels that render between your legs while worning the Assunzione robes are now hidden by tails or other clothes."
```
Comment (SleepyGemmy) [inline html/changelogs/SimpleMaroon-assunzionerobething.yml:41]:
```suggestion
  - bugfix: "Fixed Azzunzione robes showing through clothing and tails."
```

**Review by SleepyGemmy — COMMENTED** [review]
image
seems some states in the robe_backing icon are leftover?

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -765,6 +765,19 @@
 	overlay_state = "assunzione_robe"
 	contained_sprite = TRUE
 
+/obj/item/cltohing/accessory/poncho/assunzione/get_mob_overlay(var/mob/living/carbon/human/H, var/mob_icon, var/mob_state, var/slot)
```
Comment (SleepyGemmy) [inline code/modules/clothing/under/accessories/accessory.dm:768]:
```suggestion
/obj/item/clothing/accessory/poncho/assunzione/get_mob_overlay(var/mob/living/carbon/human/H, var/mob_icon, var/mob_state, var/slot)
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**